ft-sdk/migrations/0002-fastn-email-queue.sql

```diff
@@ -0,0 +1,58 @@
+CREATE TABLE "new__fastn_email_queue"
+(
+    id           INTEGER PRIMARY KEY,
+    from_address TEXT NOT NULL,
+    from_name    TEXT NOT NULL,
+    reply_to     TEXT NULL,
+    to_address   TEXT NOT NULL,
+    cc_address   TEXT NULL,
+    bcc_address  TEXT NULL,
+    subject      TEXT NOT NULL,
+    body_text    TEXT NOT NULL,
+    body_html    TEXT NOT NULL,
+    retry_count  INTEGER NOT NULL DEFAULT 0,
+    created_at   INTEGER NOT NULL,
+    updated_at   INTEGER NOT NULL,
+    sent_at      INTEGER NOT NULL,
+    mkind        TEXT NOT NULL,
+    status       TEXT NOT NULL
+) STRICT;
+
+INSERT INTO "new__fastn_email_queue"
+(
+    "id",
+    "from_address",
+    "from_name",
+    "reply_to",
+    "to_address",
+    "cc_address",
+    "bcc_address",
+    "subject",
+    "body_text",
+    "body_html",
+    "retry_count",
+    "created_at",
+    "updated_at",
+    "sent_at",
+    "mkind",
+    "status"
+)
+SELECT
+    "id",
+    "from_address",
+    "from_address",
```

Comment:
```suggestion
    '',
```

We have to add an empty name, as original from_address may contain name as per our original contract, as current code would lead to mail being sent from eg “amit <amit@amit.com> <amitu <amitu@amit.com>>"